programs/README.md

```diff
@@ -1,30 +1,34 @@
-Script 1_RS_collate_harmonize.Rmd collates and harmonizes the output from GEE, 2_LSC2_secchi_matchup.Rmd creates the matchup files, X_LS4-7_LS8-9_handoff.Rmd is a quick model to aide in more transferable data between the two atmospheric correction methods.
+# yojoa-rs-secchi/programs
 
-Script 1_RS_collate_harmonize.Rmd assumes that you have run the Landsat stack files in the 'landsat_c2' folder within this repository or have access to the Drive folder with the output created in those scripts. You will not be able to run this code if you have not completed the LS stack run or if the output has not been shared with you.
+This directory contains the scripts that collate, harmonize, and generally munge the RS, Secchi, and met data into a common file. The programs are numbered in the order that they were originally run.
```

Comment:
meteorological (met)? my mind kept seeing this and automatically thinking it was meta just mispelled

Reply:
HA! cool - I'll try to catch those and spell them out!